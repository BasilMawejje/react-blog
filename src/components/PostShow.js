import React from 'react'

class PostShow extends React.Component {
    render() {
        return (
            <div>
                Show Post
            </div>
        );
    };
}

export default PostShow;