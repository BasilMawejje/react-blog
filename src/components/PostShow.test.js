import React from 'react';
import { configure, shallow } from 'enzyme'
import Adapter from 'enzyme-adapter-react-16'
import configureStore from 'redux-mock-store';
import { Provider } from 'react-redux'
import { shallowToJson } from 'enzyme-to-json';

import PostShow from './PostShow'

configure({ adapter: new Adapter() });

const mockStore = configureStore();
const store = mockStore({posts: 
    [{ id: 1, title: 'Some post', body: 'lorem ipsum ...' }]
});

describe('PostShow', () => {
    let wrapper; 

    beforeEach(() => {
        wrapper = shallow(
            <Provider store={store}>
                <PostShow />
            </Provider>
        ).dive();
    });

    it('renders properly', () => { 
        expect(shallowToJson(wrapper)).toMatchSnapshot();
    });
})