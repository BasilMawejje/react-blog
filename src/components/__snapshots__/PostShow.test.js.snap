// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PostShow renders properly 1`] = `<Connect(PostShow) />`;
